lib/paper_trail/frameworks/active_record/models/paper_trail/version.rb

```diff
@@ -7,5 +7,12 @@ module PaperTrail
   # See the readme for details.
   class Version < ::ActiveRecord::Base
     include PaperTrail::VersionConcern
+
+    # You can now get user object (supports Device defaults)
+    def user(user_model = :User, primary_key = :id)
+      unless whodunnit.nil?
+      	user_model.to_s.constantize.find_by(primary_key.to_s.concat(' = ?'), whodunnit)
+      end
+    end
```

Comment:
👍 ... Only improvements I would throw out are renaming the argument name as I mentioned above, and also the method name, since as you mentioned, `user` may not be appropriate for all scenarios.